AutomatedTesting/project.json

```diff
@@ -14,6 +14,7 @@
     "display_name": "AutomatedTesting",
     "icon_path": "preview.png",
     "external_subdirectories": [
+        "D:/Gems/EasyTerrain",
```

Comment:
This is a hard coded absolute path, and a windows one at that. It needs to be relative